lib/api/fields/assets/js/media.js

```diff
@@ -172,15 +172,15 @@
 
 			});
 
-			this.container.on( 'click', '.dashicons-post-trash', function( e ) {
+			this.container.on( 'click', '.bs-button-trash', function( e ) {
```

Comment:
I'd suggest using additional `js-` prefixed classes so that classes used for styling does not use the same classes used as script hooks.

Reply:
IMHO Adding duplicate class attributes with a `js-` is overkill.  Class attributes exist for targeting the element.  That applies to both scripts and styles.  Why have a class attribute for a style and another one for the script target?  What purpose does that serve?

For example, here is the HTML for the above button:

```
<button aria-label="Delete Image" type="button" class="button bs-button-trash dashicons dashicons-post-trash"></button>
```

Now if we expanded and standardized on adding an additional attribute for the script, then our HTML becomes:

```
<button aria-label="Delete Image" type="button" class="button bs-button-trash dashicons dashicons-post-trash js-bs-button-trash"></button>
```

Notice the redundancy in the final HTML markup for this element.  We now have to maintain 2 separate sets of class attributes in the view file and our tests. 

IMHO, it's more complex and overkill.

BTW The dashicons class attributes will be removed in Beans 2.0.  They are there now for backwards compatibility, but are not being used in Beans.